config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users,:posts,:likes,:comments
```

Comment:
- [ ] As suggested by the previous reviewer that it would be great if you don't make unnecessary endpoints 👍 to follow the project requirements 😊 To do that you can make your endpoints something like the below ⬇️

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```